Pod/Classes/SKYPubsubContainer.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithContainer:(SKYContainer *)container
     return self;
 }
 
-- (void)configAddress:(NSString *)address
+- (void)_configAddress:(NSString *)address
```

Comment:
I think the various `_configXXX` and `configXXX` methods here are getting hard to understand and maintain, I start thinking directly drop the original `configureAddress:` and `configureWithAPIKey:` would be better, coz code migration here should be very easy.